### PR TITLE
Don't Always Push to DockerHub

### PR DESCRIPTION
**Pull request by williamblevins**
## Change Description

From [AUTO-1016](https://blockone.atlassian.net/browse/AUTO-1016), we have downstream consumers of our CI systems who may not want to push to DockerHub. This pull request allows these downstream consumers who are running our CI code to push to internal registries exclusively.

## Change Type
**Select *ONE*:**
- [ ] Documentation

- [ ] Stability bug fix

- [X] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [X] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: .cicd/helpers/general.sh [hunk 1]
```diff
@@ -7,8 +7,13 @@ export MOUNTED_DIR='/eos'
...
 export DOCKERHUB_CONTRACTS_REGISTRY="docker.io/eosio/ci-contracts-builder"
-export CI_REGISTRIES=("$DOCKERHUB_CI_REGISTRY" "$MIRROR_REGISTRY")
-export CONTRACT_REGISTRIES=("$DOCKERHUB_CONTRACTS_REGISTRY" "$MIRROR_REGISTRY")
+if [[ "$BUILDKITE_PIPELINE_SLUG" =~ "security" ]] ; then
+    export CI_REGISTRIES=("$DOCKERHUB_CI_REGISTRY" "$MIRROR_REGISTRY")
+    export CONTRACT_REGISTRIES=("$DOCKERHUB_CONTRACTS_REGISTRY" "$MIRROR_REGISTRY")
+else
+    export CI_REGISTRIES=("$MIRROR_REGISTRY")
+    export CONTRACT_REGISTRIES=("$MIRROR_REGISTRY")
+fi
```
Comment (kj4ezj) [inline .cicd/helpers/general.sh:16]: I think this logic is backwards.

**Review by kj4ezj — CHANGES_REQUESTED** [review]
🔀

File: .cicd/helpers/general.sh (hunk 1 above)
Comment (williamblevins) [reply .cicd/helpers/general.sh]: Yeah, good catch. I think CONTRACTS flags are turned off in the pipeline, so I wasn't paying enough attention ;(

**Review by kj4ezj — APPROVED** [review]
🐳

**Pull request merged**